cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/StrimziPodSetController.java

```diff
@@ -130,6 +130,7 @@ public void onDelete(Pod pod, boolean deletedFinalStateUnknown) {
         }, 10 * 60 * 1000);
 
         controllerThread = new Thread(this, "StrimziPodSetController");
+        this.workQueue = new ArrayBlockingQueue<>(podSetControllerWorkQueueSize);
```

Comment:
Actually, I think the ST failures might be related to this. I think this needs to be done first -> before all the informers etc.

Reply:
I moved it before informers now